module/documents/actor.mjs

```diff
@@ -88,8 +90,9 @@ export default class ActorEd extends Actor {
   /**
    * Legend point History earned prompt trigger
    */
-  async legendPointHistoryEarned() {
-    const historyEarned = await LegendPointHistoryEarnedPrompt.waitPrompt();
+  async legendPointHistoryEarned( actor ) {
```

Comment:
Why is the opening of the history in the document? Is there actually some post-processing planed? If so, what? If not, this is pure UI thing, so should be handled in the sheet.